CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ All user visible changes to this project will be documented in this file. This p
 
 - UI:
     - Redesigned media player. ([#1368], [#1367], [#1356])
+    - Set blockfolded whenever chat is favorite ([#1274](https://github.com/team113/messenger/issues/1274), [#1391](https://github.com/team113/messenger/pull/1391))
+        - User info page
+        - Group chat info page
```

Comment:
1.

> blockfolded

Is `blockfolded` an English word? I tried googling it, but didn't succeed.

2. You're missing a dot at the end. Take a look at another entries in the CHANGELOG file.

3.
> `([#1274](https://github.com/team113/messenger/issues/1274), [#1391](https://github.com/team113/messenger/pull/1391))`

This isn't a format that is used for links in CHANGELOG. Take a look at another entries there.

4.
https://github.com/team113/messenger/pull/1391#discussion_r2315384621:
> 3. CHANGELOG entry is a single entry, not a collection of entries formed like this.

I still see a list of entries, not a single entry.